web/massastation/src/pages/Store/StationSection/StationPlugin.tsx

```diff
@@ -73,11 +68,7 @@ export function StationPlugin({
   }
 
   const argsOn = {
-    preIcon: massalabsNomination.includes(author) ? (
-      <MassaWallet variant="rounded" />
-    ) : (
-      <img src={logo} />
-    ),
+    preIcon: <img src={logo} />,
```

Comment:
```suggestion
    preIcon: <img src={logo} alt="plugin-logo" />,
```

When you use `<img />`, don't forget to add `alt` value. 😎